api/distributed_exec.md

```diff
@@ -23,6 +23,12 @@ any introduced inconsistency.
 Note that the command is _not_ executed on the access node itself and
 it is not possible to chain multiple commands together in one call.
 
+<highlight type="important">
+You cannot run `distributed_exec` with some SQL commands. For example, `ALTER
```

Comment:
@svenklemm, are there more exceptions I should call out here?

Reply:
I'm not aware of any others.